include/alpaka/atomic/AtomicCpu.hpp

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2021 Andrea Bocci, Felice Pantaleo
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#ifndef ALPAKA_DISABLE_ATOMIC_ATOMICREF
+#    include <alpaka/atomic/AtomicAtomicRef.hpp>
+#else
+#    include <alpaka/atomic/AtomicStdLibLock.hpp>
+#endif // ALPAKA_DISABLE_ATOMIC_ATOMICREF
+
+namespace alpaka
+{
+#ifndef ALPAKA_DISABLE_ATOMIC_ATOMICREF
+    using AtomicCpu = AtomicAtomicRef;
+#else
+    using AtomicCpu = AtomicStdLibLock<16>;
+#endif // ALPAKA_DISABLE_ATOMIC_ATOMICREF
+
+} // namespace alpaka
```

Comment:
There is a missing newline here.

Reply:
would something like this work?
 `The atomic ops based on atomic_ref for CPU accelerators.`